_docs/guide/developers/scripting-tester.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Automation Scripting Tester
+excerpt: This page shows how to access and use the Automation Scripting Tester in the developer menu.
+permalink: /docs/guide/developers/scripting-tester/
+toc:
+  title: Automation Scripting
+  expand: Developer Guide
+jumbotron:
+  title: Automation Scripting Tester
+  breadcrumbs:
+  - label: Docs &raquo;
+    url: /docs/home/
+  - label: Developer Guide &raquo;
+---
+
+The Automation Scripting Tester is perfect for writing one-off scripts that don't warrant an entire automation. 
+
+## Access
+
+The Data Query Tester can be accessed by clicking **Setup > Developers > Automation Scripting Tester.** 
```

Comment:
This is the automation scripting tester page, but it says "Data Query Tester" here.